lib/Agrammon/UI/CommandLine.pm6

```diff
@@ -46,34 +47,76 @@ multi sub MAIN('web', ExistingFile $cfg-filename, ExistingFile $model-filename,
 
 #| Run the model
 multi sub MAIN('run', ExistingFile $filename, ExistingFile $input, Str $technical-file?,
-               SupportedLanguage :$language = 'de', Str :$prints, Str :$variants = 'SHL',
+               SupportedLanguage :$language = 'de', Str :$prints, Str :$variants = 'Base',
                Bool :$include-filters, Bool :$include-all-filters=False, Int :$batch=1, Int :$degree=4, Int :$max-runs,
                OutputFormat :$format = 'text'
               ) is export {
-    my %results = run $filename.IO, $input.IO, $technical-file, $variants, $format, $language, $prints,
+    my $data = run $filename.IO, $input.IO, $technical-file, $variants, $format, $language, $prints,
             ($include-filters or $include-all-filters),
             $batch, $degree, $max-runs, :all-filters($include-all-filters);
+    my %results = $data.results;
+    my %validation-errors = $data.validation-errors;
```

Comment:
Potentially `:=` here to avoid some copying

Reply:
Done, same for `%results`